app/models/member.rb

```diff
@@ -25,6 +25,7 @@ class Member < ActiveRecord::Base
   has_one :braintree_customer,     class_name: 'Payment::Braintree::Customer'
   has_one :authentication, class_name: MemberAuthentication, dependent: :destroy
   has_many :payment_methods, through: :customer
+  has_many :actions
```

Comment:
Yes, that's the case. I needed `member.actions` for `member.actions.last.page.language.code`. Would you like me to remove this, or is it good to leave?